release/pkg/bundles/package-controller.go

```diff
@@ -87,17 +87,18 @@ func GetPackagesBundle(r *releasetypes.ReleaseConfig, imageDigests map[string]st
 				var digest string
 				digest = imageDigests[imageArtifact.ReleaseImageURI]
 
-				if !r.DevRelease {
-					requires, err := helm.GetChartImageTags(driver, helmdir)
-					if err != nil {
-						return anywherev1alpha1.PackageBundle{}, errors.Wrap(err, "Error retrieving requires.yaml")
-					}
-					for _, images := range requires.Spec.Images {
-						if images.Repository == imageArtifact.AssetName {
-							digest = images.Digest
-						}
-					}
-				}
```

Comment:
Isn't the bundle going to be missing the image list?

Reply:
It might look weird in the GH editor, This bracket is tied to the top level one. 
<img width="579" alt="Screen Shot 2022-09-29 at 11 16 35 AM" src="https://user-images.githubusercontent.com/34656423/193084579-0a089531-2eb7-4ab8-bac0-1fba9b858a4f.png">